### DISCUSSION TITLE
Integration not listed when trying to add it

### DISCUSSION THREAD

**Discussion by enc1976** [astrandb/miele#108] [General]
Hi and many thanks for your wonderful contribution! I am facing a little problem - I have installed the integration via hacs and also added the config info in the yaml. Nevertheless, when I am trying to add via "add new integration" it does not appear in the list. What am I doing wrong? 

Many thanks!

**Comment by astrandb** [comment]
Restart HA after adding the integration via HACS and editing configuration yaml. Sometimes you have to refresh browser cache also.

**Comment by enc1976** [comment]
Hi! Many thanks, it worked! Maybe it would be useful to propose the browser refresh in the documentation - it might be useful for other people as well.

Btw, the integration works very well! Congrats! I will open a separate thread for my ovens.

**Reply by astrandb** [reply]
I added a few words in the docs. However, this is an issue with HA or HACS, not related to this specific integration.
